spec/features/form/validations_spec.rb

```diff
@@ -37,7 +37,7 @@
 
   describe "Question validation" do
     context "when the tenant age is invalid" do
-      it "shows validation for under 0" do
+      it "shows validation for under 0", js: true do
```

Comment:
Does this need js or was this just for debugging?

Reply:
I suspect this was left over from debugging, nice catch 🙂